src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,30 @@
+$card-width: 200px;
+
+.catalog {
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
```

Comment:
I suppose, you should delete **margin-bottom: 48px**

Reply:
Agree